.env.template

```diff
@@ -105,9 +105,9 @@ TRTL_REPLICA_GOSSIP_SIGMA=1500ms
 
 # Trtl: Replica Configuration Strategy
 TRTL_REPLICA_STRATEGY_HOSTNAME_PID=false
-TRTL_REPLICA_STRATEGY_HOSTNAME=
-TRTL_REPLICA_STRATEGY_FILE_PID=false
-TRTL_REPLICA_STRATEGY_JSON_CONFIG=
+TRTL_REPLICA_STRATEGY_HOSTNAME=""
+TRTL_REPLICA_STRATEGY_FILE_PID=""
+TRTL_REPLICA_STRATEGY_JSON_CONFIG=""
```

Comment:
Is this a functional change or a cosmetic one?

Reply:
It's hard to say - I was getting an error that it couldn't load the PID file, and when I added the quotes it seemed to fix it; however I didn't have that problem before, so I'm not sure what is going on. I couldn't reproduce just now - so it must be unrelated.